package.json

```diff
@@ -20,27 +20,28 @@
   "author": "Phrase",
   "license": "MIT",
   "devDependencies": {
-    "@types/jest": "^29.5.4",
-    "@types/node": "^20.6.0",
-    "@types/react": "^18.2.21",
-    "@types/react-dom": "^18.2.7",
-    "@types/react-test-renderer": "^18.0.1",
-    "jest": "^29.7.0",
-    "jest-environment-jsdom": "^29.7.0",
-    "react": "^18.2.0",
-    "react-addons-test-utils": "^15.6.2",
-    "react-dom": "^18.2.0",
-    "react-intl": "^6.4.6",
-    "react-test-renderer": "^18.2.0",
-    "ts-jest": "^29.1.1",
-    "ts-loader": "^9.4.4",
-    "typescript": "^5.2.2",
-    "webpack": "^5.88.2",
-    "webpack-cli": "^5.1.4"
+    "@testing-library/dom": "^10.4.1",
+    "@testing-library/react": "^16.3.0",
+    "@types/jest": "^30.0.0",
+    "@types/node": "^24.3.0",
+    "@types/react": "^19.1.10",
+    "@types/react-dom": "^19.1.7",
+    "@types/react-test-renderer": "^19.1.0",
+    "jest": "^30.0.5",
+    "jest-environment-jsdom": "^30.0.5",
+    "react": "^19.1.1",
+    "react-dom": "^19.1.1",
+    "react-intl": "^7.1.11",
+    "react-test-renderer": "^19.1.1",
+    "ts-jest": "^29.4.1",
+    "ts-loader": "^9.5.2",
+    "typescript": "^5.9.2",
+    "webpack": "^5.101.2",
+    "webpack-cli": "^6.0.1"
   },
   "peerDependencies": {
-    "react": ">=18.0.0",
-    "react-intl": ">=6"
+    "react": ">=19.0.0",
+    "react-intl": ">=7"
```

Comment:
no sure how currently we deal with the bumping of these versions (react and react-intl)